.github/workflows/cross.yml

```
@@ -339,3 +341,47 @@ jobs:
       - name: Build
         run: |
           docker run -v "${{ github.workspace }}:/aws_lc_rs" alpine:3.20
+
+  msys2-clang:
+    if: github.repository_owner == 'aws'
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { sys: mingw64, env: x86_64, target: x86_64-pc-windows-gnu, cc: gcc, cxx: g++ }
+          - { sys: mingw32, env: i686, target: i686-pc-windows-gnu, cc: gcc, cxx: g++ }
+          - { sys: ucrt64,  env: ucrt-x86_64,  target: x86_64-pc-windows-gnu, cc: gcc, cxx: g++ }
+          - { sys: clang64, env: clang-x86_64, target: x86_64-pc-windows-gnullvm, cc: clang, cxx: g++ }
+    steps:
+      - name: Install MSYS2
+        uses: msys2/setup-msys2@v2
+        id: setup_msys2
+        with:
+          msystem: ${{ matrix.sys }}
+          update: true
+          pacboy: >-
+            clang
```

```suggestion
            cc
            rust-bindgen
```
cc is a meta package which points to main C compiler of the environment. as rust-bindgen is used during the build, better use one from MSYS2

---

oh I see it's a cross compilation. you may not add rust-bindgen then